index.js

```diff
@@ -32,6 +33,19 @@ export async function transformSource(source, context) {
   return { source: result };
 }
 
+// Node >= v16.12.0:
+export async function load(sourceUrl, context, defaultLoad) {
+  if (!isJsModule(sourceUrl) || isPackageModule(sourceUrl)) {
+    return defaultLoad(sourceUrl, context, defaultLoad)
```

Comment:
Doesn't this create a loop, passing the `defaultLoad` as a callback to the `defaultLoad`?

Reply:
No, I'm calling the defaultLoad function instead of using custom load logic (for all files that are not relevant)